src/commands/run-build.yml

```diff
@@ -10,12 +10,18 @@ parameters:
       in the plan returned by buildfile.m as well as all the tasks on which they depend.
     type: string
     default: ""
-  startup-options:
+  build-options:
     description: >
-      MATLAB startup options. If you specify more than one option, use a space to
+      Build options for MATLAB build tool. If you specify more than one option, use a space to
```

Comment:
Hi, @mw-kapilg. Please reuse this text for parameter descriptions: https://github.com/matlab-actions/run-build?tab=readme-ov-file#run-matlab-build

Reply:
Hey @mw-hrastega , thanks for catching it early. I'll fix it in the new PR, please ignore this closed PR.